src/applications/search/containers/SearchApp.jsx

```diff
@@ -51,13 +57,19 @@ class SearchApp extends React.Component {
     const { userInput, page } = this.state;
     if (userInput) {
       this.props.fetchSearchResults(userInput, page);
-      this.writeBreadcrumb();
     }
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.search.query !== prevProps.search.query) {
-      this.writeBreadcrumb();
+    const hasNewResults =
+      prevProps.search.loading && !this.props.search.loading;
+
+    if (hasNewResults) {
+      const shouldFocusOnResults = this.props.search.searchesPerformed > 1;
+
+      if (shouldFocusOnResults) {
+        focusElement(`.${SCREENREADER_FOCUS_CLASSNAME}`);
+      }
```

Comment:
This reads as -

1. If the app was previously loading (`prevProps.search.loading`) and is not loading now (`!this.props.search.loading`), then we have new results
2. If we have new results, and this is not the first search, then we should focus on the result set
3. Focus!